Packs/LinuxEventsCollection/README.md

```diff
@@ -1,52 +1,75 @@
+2. Install the XDR Collector installation package for Linux on the requested Linux servers as described [here](https://docs-cortex.paloaltonetworks.com/r/Cortex-XSIAM/Cortex-XSIAM-Documentation/Install-the-XDR-Collector-installation-package-for-Linux). 
+3. Configure an [XDR Collector Filebeat profile](https://docs-cortex.paloaltonetworks.com/r/Cortex-XSIAM/Cortex-XSIAM-Documentation/XDR-Collector-profiles) for the requested linux servers as described [here](https://docs-cortex.paloaltonetworks.com/r/Cortex-XSIAM/Cortex-XSIAM-Documentation/Add-an-XDR-Collector-profile-for-Linux).
+      - When configuring the Filebeat YAML configuration file, use the *LinuxEventCollection* template as a reference:
+ ![LinuxEventCollection Filebeat Template](https://raw.githubusercontent.com/demisto/content/435111554ddbc4b737b48688314aa466e4a4b8a2/Packs/LinuxEventsCollection/doc_files/LinuxFilebeatTemplate.png)
+    - Customize the *[paths](https://www.elastic.co/guide/en/beats/filebeat/current/filebeat-input-filestream.html#filestream-input-paths)* parameter in accordance to the requested log files for collection: 
+         ```yaml
+            filebeat.inputs:
+            - type: filestream
+              enabled: true
+              paths: # customize paths as necessary 
+                - /var/log/syslog
+                - /var/log/auth.log
+                - /var/log/messages
+                - /var/log/secure
+              processors:
+                - add_fields:
+                    fields:
+                      vendor: linux
+                      product: linux
+         ```
+4. Apply the configured Filebeat profile to the requested target Linux servers by attaching it to a policy as described [here](https://docs-cortex.paloaltonetworks.com/r/Cortex-XSIAM/Cortex-XSIAM-Documentation/Apply-profiles-to-collection-machine-policies).
+
+
+#### Supported Timestamp formats 
+##### Broker VM Syslog
+- [RFC 3164](https://datatracker.ietf.org/doc/html/rfc3164#section-4.1.2) compatible timestamps, in UTC, for example: *`Oct  8 19:44:40`*. Note: You can run the following command on the relevant Linux servers for configuring their timezone to UTC:       
+  ```bash 
+  sudo timedatectl set-timezone UTC
+  ```
+- [RFC 3339](https://www.rfc-editor.org/rfc/rfc3339) compatible like timestamps. Examples: 
+   - *`2024-10-28T14:30:55Z`*
+   - *`2024-10-28T14:30:55-0300`*
+   - *`2024-10-28T14:30:55.123+02:00`*
+  
+
+##### Filebeat XDR Collector
+- [RFC 3339](https://www.rfc-editor.org/rfc/rfc3339) compatible like timestamps. Examples: 
+   - *`2024-10-28T14:30:55Z`*
+   - *`2024-10-28T14:30:55-0300`*
+   - *`2024-10-28T14:30:55.123+02:00`*  
+
+**Note**: Events logs collected via Filebeat with timestamp formats other than those listed above would be assigned with the agent collection time.
```

Comment:
Recommend:
new line for note as follows
**Note:**
Event logs collected via Filebeat with timestamp formats other than those listed above are assigned the agent collection time.

Reply:
Fixed:
https://github.com/demisto/content/pull/36882/commits/be339141a34bd645960ec4e017b1ab64e68a7424#diff-547b4347d8e4a399e431acab3ca503579d3d23c0f8854847ee85c97343769646R79